### PR TITLE
Introducing new design using Jekyll & Bootstrap

### PR DESCRIPTION
**Pull request by guillaumebriday**
## Migrate from Middleman to Jekyll for better support

Adding .ruby-version
Converting sass to scss
Converting haml to html
Removing useless assets
Removing useless or obsolete pages
Removing useless sitemap
Update robots.txt

Also here's a new design! 💅

## Before

Screenshot 2024-10-17 at 22-03-42 Lyon rb Lyon Ruby Brigade

## After

Screenshot 2024-10-17 at 21-47-21 Lyon rb

### CODE REVIEW

File: _includes/nav.html [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-      <a href="https://www.meetup.com/LyonRB">
-        <i class="fab fa-meetup"></i> S'inscrire à nos meetups
+      <a href="{{ site.youtube_url }}" class="btn btn-outline-primary" target="_blank">
+        <i class="fab fa-youtube"></i>
+
+        Voir nos replays
       </a>
-    </li>
-  </ul>
+    </div>
```
Comment (guillaumebriday) [inline _includes/nav.html:19]: Rajouter lien : https://forms.gle/gA2Cko64qP6kLQvX8

**Comment by cappert** [comment]
Merged the two PRs because it was easier to test that way.

**Comment by guillaumebriday** [comment]
> Merged the two PRs because it was easier to test that way.

Yep perfect, it was to ease the review! 👍

File: .github/workflows/jekyll.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+jobs:
+  # Build job
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
```
Comment (cappert) [inline .github/workflows/jekyll.yml:37]: we might want to use a commit hash (to lock ruby versions?)
The sample workflow Github provided had an very old hash so I used the v1 version which is basically "latest"

**Review by cappert — APPROVED** [review]
added a Github Actions workflow and tested it in my fork, it should pick up the CNAME automatically.

Ran a [Lighthouse automated audit](https://developer.chrome.com/docs/lighthouse/overview) and got very good results (except from the Youtube embed but we can't do anything about that).

Love the new design !

**Pull request merged**